src/test/java/ConsoleDisplayTest.java

```diff
@@ -58,14 +58,35 @@ public void askGameMode() {
                 "play against the computer or '2' to play human-vs-human.\n", ioHelper.output());
     }
 
+    @Test
+    public void announceInputInvalid() {
+        IOHelper ioHelper = new IOHelper("");
+        ConsoleDisplay consoleDisplay = new ConsoleDisplay(ioHelper.in, ioHelper.print);
+
+        consoleDisplay.announceInputInvalid();
+
+        assertEquals("\nLooks like you made a boo-boo! Please enter a number from 1-9 that hasn't already been picked.\n", ioHelper.output());
```

Comment:
Some more important vocab for your German language efforts: "Bubu machen" means "to take a nap" in German 😴